insalan/tournament/serializers.py

```diff
@@ -54,8 +54,9 @@ class Meta:
         """Meta options of the serializer"""
 
         model = Tournament
-        read_only_fields = ("id",)
-        fields = ["id", "event", "game", "name", "teams", "logo"]
+        read_only_fields = ("id","manager_price_onsite", "manager_price_onsite", 
```

Comment:
cashprize is readonly I think

Reply:
Unsure, it should be editable in case of new partnership increasing the cashprize I guess?

Reply:
isn't is for the fetching endpoint ?

Reply:
These are serializers, access rights should be managed in views (for example a `ReadOnly | IsAdmin`), however since we've put prices and all read-only, I guess we can do the same with the cashprize to be sure :woman_shrugging:

Reply:
I guess you're right

Reply:
Yeah I'll set them we read-only on serialization and editable in admin view

Reply:
Wait actually everything should be read-only?

Reply:
Indeed